string IP
int32 isperson
---
int32 player_id

